### PR TITLE
oskar/response_headers

### PR DESCRIPTION
**Pull request by oskar-binary**
Update Respone header cache-control handling as per https://www.gatsbyjs.org/docs/caching/

### CODE REVIEW

File: deriv.com.conf [hunk 1]
```diff
@@ -32,7 +32,26 @@ server {
...
-    add_header Cache-Control "public, max-age=7200, s-maxage=600";
+    location ~* \.html$ {
+        add_header Cache-Control "public, no-cache, must-revalidate";
+    }
+
+    location /page-data {
+        add_header Cache-Control "public, no-cache, must-revalidate";
+    }
+
+    location = /sw.js {
```
Comment (mustofa-binary) [inline deriv.com.conf:43]: is sw.js service worker related?

Comment (oskar-binary) [reply deriv.com.conf:43]: yes we never want to cache the service worker - as we it should be revalidated upon each load to check if a new version of the site is available. (once we add the service worker that is :) )

File: deriv.com.conf [hunk 2]
```diff
@@ -32,7 +32,26 @@ server {
...
+        add_header Cache-Control "public, no-cache, must-revalidate";
+    }
+
+    location /static {
+        add_header Cache-Control "public, max-age=31536000, immutable";
```
Comment (mustofa-binary) [inline deriv.com.conf:48]: any reason why keep cache for 1 year?, isnt it too long, why dont put it a day or a week then it will be revalidated?

Comment (oskar-binary) [reply deriv.com.conf:48]: These files get a unique hash from webpack (from content) so they are safe to cache forever: check https://www.gatsbyjs.org/docs/caching/

Comment (mustofa-binary) [reply deriv.com.conf:48]: nice!, 👍

**Pull request merged**